olive/model/utils/hf_utils.py

```diff
@@ -87,10 +87,10 @@ def patched_supported_features_mapping(*supported_features: str, onnx_config_cls
     if onnx_config_cls is None:
         raise ValueError("A OnnxConfig class must be provided")
 
-    import olive.model.utils.hf_onnx_config as config_cls
+    from olive.model.utils import hf_onnx_config
 
     for attr_name in onnx_config_cls.split("."):
```

Comment:
oh, now I remember why I did the hacky import thing. I wanted to support getting attributes like `a.b.c`. 

I already added a utils functions to do this. 
Can you replace lines 92-93 with `onnx.common.utils.get_attr(hf_onnx_config, attr_name)`. This way the recursive `getattr` behavior is same as before.

Reply:
will do